src/applications/discharge-wizard/components/InstructionsPage.jsx

```diff
@@ -28,353 +28,350 @@ class InstructionsPage extends React.Component {
 
   render() {
     return (
-      <div className="dw-instructions">
-        <h1>How to Apply for a Discharge Upgrade</h1>
-        <div itemScope itemType="http://schema.org/FAQPage">
-          <div className="row">
-            <article className="usa-content columns">
-              <div className="va-introtext">
-                <p>
-                  Answer a series of questions to get customized step-by-step
-                  instructions on how to apply for a discharge upgrade or
-                  correction. If your application goes through and your
-                  discharge is upgraded, you’ll be eligible for the VA benefits
-                  you earned during your period of service.
-                </p>
-              </div>
-              <div className="main home signup">
-                <div className="section main-menu">
-                  <div className="row">
-                    <div className="small-12 columns">
-                      <div>
-                        <p>
-                          All branches of the military consider you to have a
-                          strong case for a discharge upgrade if you can show
-                          your discharge was connected to any of these
-                          categories:
-                        </p>
-                        <ul>
-                          <li>
-                            Mental health conditions, including posttraumatic
-                            stress disorder (PTSD)
-                          </li>
-                          <li>Traumatic brain injury (TBI)</li>
-                          <li>
-                            Sexual assault or harassment during military service
-                            (at VA, we refer to this as military sexual trauma
-                            or MST)
-                          </li>
-                          <li>
-                            Sexual orientation (including under the Don’t Ask,
-                            Don’t Tell policy)
-                          </li>
-                        </ul>
-                        <p>
-                          The information you enter on the next page is
-                          completely confidential.
-                        </p>
-                        <p>
-                          <Link
-                            className="usa-button-primary va-button"
-                            to="questions"
-                          >
-                            Get started »
-                          </Link>
-                        </p>
-                      </div>
-                    </div>
+      <div
+        className="dw-instructions"
+        itemScope
+        itemType="http://schema.org/FAQPage"
```

Comment:
Sorry for all the changes, I think the whole HTML indenting had to update when I saved! I'll go ahead and highlight the most important LOC that I changed.

I moved this itemType up one div so that we could use the <h1> as our `name`.

Reply:
it's no problem. GitHub PR viewer has a setting to hide the whitespace changes. 
<img width="246" alt="Screen Shot 2019-09-23 at 2 53 52 PM" src="https://user-images.githubusercontent.com/4998130/65461956-07042300-de12-11e9-8653-758edadbbb87.png">